chameleon-core/src/test/java/org/hibernate/omm/crud/SimpleCRUDTests.java

```diff
@@ -120,6 +120,34 @@ void testUpdate() {
         });
     }
 
+    @Test
+    void testUpdateQuery() {
+        insertBook();
+        var newAuthor = "Fyodor Dostoevsky";
+        var newTitle = "Crime and Punishment";
+        int newPublishYear = 1866;
+
+        // the following JSON command will be issued:
+        // { update: "books", updates: [ { q: { _id: { $eq: ? } }, u: { $set: { author: ?, publishYear: ?, title: ? } } } ] }
+        sessionFactory.inTransaction(session -> {
+            var update = session.createMutationQuery("update Book " +
+                    "set author = :author, title = :title, publishYear = :publishYear " +
+                    "where id = :id");
+            update.setParameter("id", id);
+            update.setParameter("author", newAuthor);
+            update.setParameter("title", newTitle);
+            update.setParameter("publishYear", newPublishYear);
+            update.executeUpdate();
```

Comment:
It would be more idiomatic to chain the parameter setting methods and even the last `executeUpdate()` method:
```
update.setParameter("id", id)
            .setParameter("author", newAuthor)
            .setParameter("title", newTitle)
            .setParameter("publishYear", newPublishYear)
            .executeUpdate();
```